app/controllers/all/export/grant_management_and_finance_unit/transfers/projects_controller.rb

```diff
@@ -0,0 +1,42 @@
+class All::Export::GrantManagementAndFinanceUnit::Transfers::ProjectsController < ApplicationController
+  def index
+    authorize Project, :index?
+
+    months = []
+    export_months.each do |month|
+      months << {
+        month: month,
+        count: ProjectsForExportService.new.grant_management_and_finance_unit_transfer_projects(month: month.month, year: month.year).count
+      }
+    end
+    @data = months
+  end
+
+  def show
+    authorize Project, :show?
+    @month = Date.parse("#{year}-#{month}-1")
```

Comment:
Avoid string parsing with `Date.new` instead perhaps?

```suggestion
    @month = Date.new(year.to_i, month.to_i)
```